src/datasets/packaged_modules/__init__.py

```diff
@@ -12,9 +12,9 @@
 def hash_python_lines(lines: List[str]) -> str:
     filtered_lines = []
     for line in lines:
-        line.replace("\n", "")  # remove line breaks, white space and comments
-        line.replace(" ", "")
-        line.replace("\t", "")
+        line = line.replace("\n", "")  # remove line breaks, white space and comments
+        line = line.replace(" ", "")
+        line = line.replace("\t", "")
```

Comment:
I would just remove this replacements.
They could create some errors (return the same hash while the code is different). And since it would change the hashes, it means that all the cache directories that have been created until now would now be ignored. 

```suggestion
```